models/issue_mail.go

```diff
@@ -95,7 +135,7 @@ func (issue *Issue) MailParticipants() (err error) {
 		return fmt.Errorf("UpdateIssueMentions [%d]: %v", issue.ID, err)
 	}
 
-	if err = mailIssueCommentToParticipants(issue, issue.Poster, mentions); err != nil {
+	if err = mailIssueActionToParticipants(issue, issue.Poster, mentions); err != nil {
```

Comment:
here you'd just call `mailIssueCommentToParticipants(issue, issue.Poster, nil, mentions)` instead